exist-core/src/main/java/org/exist/repo/ExistRepository.java

```diff
@@ -286,11 +286,11 @@ public static Path getRepositoryDir(final Configuration config) throws IOExcepti
     private static void moveOldRepo(final Optional<Path> home, final Path newRepo) {
         final Path repo_dir = home.map(h -> {
             if(FileUtils.fileName(h).equals("WEB-INF")) {
-                return h.resolve(EXPATH_REPO_DIR);
+                return h.resolve(EXPATH_REPO_DIR_NAME);
             } else {
-                return h.resolve( EXPATH_REPO_DEFAULT);
+                return h.resolve(LEGACY_DEFAULT_EXPATH_REPO_DIR);
             }
-        }).orElse(Paths.get(System.getProperty("java.io.tmpdir")).resolve(EXPATH_REPO_DIR));
+        }).orElse(Paths.get(System.getProperty("java.io.tmpdir")).resolve(EXPATH_REPO_DIR_NAME));
```

Comment:
a directory in the tmp directory..... can be removed under the hood; is that bad?

Reply:
Probably, but UI haven't changed the existing behaviour here. I just renamed the variables